proposals/lip-0045.md

```diff
@@ -874,6 +849,15 @@ def sendInternal(sendingAddress: Address,
         data = {"ccm": ccm},
         topics = [ccm.sendingChainID, ccm.receivingChainID, ccmID]
     )
+
+def emitCCMFailedEvent(ccm: CCM, code: uint32):
```

Comment:
the name suggests that the execution of the ccm failed. Maybe `emitCCMSentFaildEvent`? Or `emitCCMSentFaild` if it's too long.

Reply:
Done. Used `emitCCMSentFailedEvent`.